app/routes/router.$.tsx

```diff
@@ -1,16 +1,16 @@
-import type { LoaderFunction } from '@remix-run/node'
 import { redirect } from '@remix-run/node'
+import type { LoaderFunctionArgs } from '@remix-run/node'
 
-export const loader: LoaderFunction = (context) => {
+export const loader = (context: LoaderFunctionArgs) => {
   handleRedirects(context)
 
   return redirect('/router/latest')
 }
 
-function handleRedirects(context: Parameters<LoaderFunction>[0]) {
+function handleRedirects(context: LoaderFunctionArgs) {
   const url = new URL(context.request.url)
   // prettier-ignore
-  const reactLocationv2List = [
+  const reactLocationV2List = [
```

Comment:
I think this was copied here from `app/routes/virtual.$.tsx`, because the list matches the list we have there and Router doesn't even have a v2. 

So this whole `handleRedirects` function could be deleted at this point.

Reply:
Oops you're right, I copied from the wrong spot!

Reply:
Wait no I copied the right thing back, it's just that the original wasn't functional. Again, I might fix this in a subsequent PR?

Reply:
That's cool with me! 👍